coalib/bearlib/abstractions/Linter.py

```diff
@@ -699,10 +699,12 @@ def linter(executable: str,
         ``coalib.results.RESULT_SEVERITY`` for a result. Severity strings are
         mapped **case-insensitive**!
 
-        - ``RESULT_SEVERITY.MAJOR``: Mapped by ``error``.
-        - ``RESULT_SEVERITY.NORMAL``: Mapped by ``warning`` or ``warn``.
-        - ``RESULT_SEVERITY.MINOR``: Mapped by ``info``.
-
+        - ``RESULT_SEVERITY.MAJOR``: Mapped by ``critical`` or ``c``.
+        - ``RESULT_SEVERITY.MAJOR``: Mapped by ``fatal``, ``fail`` or ``f``.
+        - ``RESULT_SEVERITY.MAJOR``: Mapped by ``error``, ``err`` or ``e``.
```

Comment:
Yeah that's what I tried to do at first, but got dinged on the line length.  I'll update again...